lib/tasks/spotlight_tasks.rake

```diff
@@ -69,6 +69,16 @@ namespace :spotlight do
     end
   end
 
+  desc 'Migrate Pages to a scoped language'
+  task :migrate_pages_language, [:language] => :environment do |_, args|
+    if args[:language]
+      require 'migration/page_language'
+      Migration::PageLanguage.run args[:language]
+    else
+      STDERR.puts "\nUsage: rake spotlight:migrate_pages_language[language]\n\n Example: rake spotlight:migrate_pages_language[en]\n\n"
```

Comment:
Would there be any use/value to us falling back to `I18n.default_locale` in this case?